readme.md

```diff
@@ -58,17 +63,17 @@ For updates on the progress see the roadmap progress [thread](https://github.com
 
 The current and key areas of development are as follows:
 
-1. improve tests - clarify the language specification and memorialize it in tests.
+1. improve tests - memorialize the slim core's it in tests.

```

Comment:
This sentence feel off. I'm not sure what the intended meaning is.

Replacing "memorialize" with a more common word or phrase might also be a good idea (in order to make the readme more accessible)